src/rubintv/app.py

```diff
@@ -104,17 +112,39 @@ def _get_blobs(self) -> List[Bucket]:
                 prefix = channel.prefix
                 print(f"Trying prefix: {prefix}")
                 blobs += list(self._bucket.list_blobs(prefix=prefix))
-                print(f"blobs found: {len(blobs)}")
+                print(f"Total blobs found: {len(blobs)}")
         return blobs
 
     def reload(self) -> None:
+        """Reloads the historical data cache"""
         self._events = self._get_events(reset=True)
         self._lastCall = get_current_day_obs()
         return
 
     def _get_events(
         self, reset: bool = False
     ) -> Dict[str, Dict[str, List[Event]]]:
+        """Returns a dict of dicts of sorted lists of Events
+
+        Either simply returns the cache of events or populates the cache before returning it.
+        It will (re)populate the cache if any of the following are True:\n
+        - There is no existing cache\n
+        - The day has rolled over\n
+        - reset is set to True
```

Comment:
```suggestion
        It will (re)populate the cache if any of the following are True:

        - There is no existing cache
        - The day has rolled over
        - reset is set to True
```